### PR TITLE
Journey - Source Page

### PR DESCRIPTION
**Pull request by amymok**
### Additions
* Added source page

### Reviews:
* @cfarm @virginiacc I need help reviewing the html classes, if there's a better way to do them then what I have now
* Will add designers to review once it looks better.

### Checklist

* [x] Changes are limited to a single goal (no scope creep)
* [x] Code can be automatically merged (no conflicts)
* [x] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [x] Visually tested in supported browsers and devices 

Windows 7
* [x] IE 11
* [x] IE 10
* [x] IE 9
* [x] IE 8
* [x] latest Chrome
* [x] latest Firefox

Mac
* [x] latest Chrome
* [x] latest Firefox
* [x] latest Safari

Mobile
* [x] latest iPhone
* [x] latest iPad
* [x] latest Android
* [ ] latest BlackBerry

### CODE REVIEW

**Comment by amymok** [comment]
### Screenshots:
@stephanieosan @huetingj 
@cfarm @virginiacc I noticed the link doesn't wrap if it is too long, how do we handle that?

Desktop:

screen shot 2015-08-12 at 6 43 50 pm
screen shot 2015-08-12 at 6 44 08 pm
screen shot 2015-08-12 at 6 44 19 pm
screen shot 2015-08-12 at 6 44 34 pm
screen shot 2015-08-12 at 6 44 45 pm

Mobile:
screen shot 2015-08-12 at 6 45 35 pm
screen shot 2015-08-12 at 6 45 45 pm
screen shot 2015-08-12 at 6 45 54 pm
screen shot 2015-08-12 at 6 46 06 pm

**Comment by cfarm** [comment]
Here are some CSS rules that help links/text wrap properly: https://css-tricks.com/snippets/css/prevent-long-urls-from-breaking-out-of-container/

**Comment by amymok** [comment]
@cfarm Thanks, I was just looking at that too :).  Fixed.
screen shot 2015-08-13 at 12 06 35 pm

**Comment by cfarm** [comment]
This looks great! Something we haven't been doing that I'm going to do for my PRs from now on is adhere to the definition of done (GHE wiki) for browser qa and testing as part of the sprint issue getting resolved. Can you also do QA on these pages using Sauce Labs?

Also, would this page be a good candidate for applying the 404 form explainer URLs browser tests functionality to the links here?

**Comment by amymok** [comment]
@cfarm Added browser tests for URLs

**Comment by huetingj** [comment]
This looks great! Only one thing: with fonts dropping down a size on mobile, the action step and bit of information become the same size. Dropping the bit of information down a size would make it a header 5, which we don't want since that's in all caps. The only other possibility I can see is making it bolded body copy (so changing it's size from 18 to 16). I'm ok leaving as is; what do you think @stephanieosan?

**Comment by stephanieosan** [comment]
I had the same reaction. If it's not too much work, I think it would be better to drop the Quote to 16px (and is it demibold or medium?).

**Comment by huetingj** [comment]
Demibold and 16px!

**Comment by amymok** [comment]
How about now? @huetingj @stephanieosan 
screen shot 2015-08-14 at 12 30 00 pm
screen shot 2015-08-14 at 12 30 10 pm

**Comment by stephanieosan** [comment]
:+1: Looks great! Let's do this!

**Comment by huetingj** [comment]
Yay!

File: test/browser_testing/features/journey.feature [hunk 1]
```diff
@@ -16,3 +16,13 @@ Examples:
   | Explore Loan Options  |
   | Compare Loan Options  |
   | Get Ready to Close    |
+
+@check_urls @amy
+Scenario Outline: Testing availability of pages that don't are not part of journey steps
```
Comment (cfarm) [inline test/browser_testing/features/journey.feature]: typo here with 'don't are not part'

**Comment by cfarm** [comment]
the new test is working :+1:

**Pull request merged**